scripts/install-windows-prereqs.ps1

```diff
@@ -2,6 +2,9 @@
 # Licensed under the MIT License.
 
 Param(
+    [Parameter(mandatory=$true)][string]$InstallPath,
+    [Parameter(mandatory=$true)][ValidateSet("yes", "no")][string]$WithFLC,
+    [Parameter(mandatory=$true)][ValidateSet("azure")][string]$WithAzureDCAPClient,
```

Comment:
I have changes that refactor these a bit here (still subject to review/approval):  https://github.com/openenclave/openenclave/pull/2122/files#diff-846bcbf4c48fa80e76202a5e31d77476R31 . Feel free to leave your changes in your PR for now, but don't worry about updating docs for the modified parameters in your PR, I'll make sure to do that in my PR!